libnethack/dat/medusa.des

```diff
@@ -48,7 +49,7 @@ DOOR:locked,(38,08)
 DOOR:locked,(38,11)
 DOOR:closed,(30,12)
 # Branch, not allowed inside Medusa's building.
-BRANCH:levregion(01,00,79,20),(30,06,46,13)
+BRANCH:levregion(00,00,78,20),(30,06,46,13)
```

Comment:
Same as before, I had to decrement this number. Please confirm the level is still sane.